bot.py

```diff
@@ -139,60 +155,92 @@ def next_in_queue(self, _):
             self.voice_client.stop()
             self.voice_client.play(self.song_queue.get(), after=self.next_in_queue)
 
+    async def get_voice_channel(self, message):
+        """
+        Get voice channel for message author. Complain if author is not in a channel
+        """
+        assert isinstance(message, discord.Message)
```

Comment:
Log error then return None instead of raising an assertion exception since this is used at serving time?